linera-base/src/identifiers.rs

```diff
@@ -59,6 +59,15 @@ impl AccountOwner {
             AccountOwner::Address20(_) => 20,
         }
     }
+
+    /// Gets the EVM address if possible
+    #[cfg(with_revm)]
+    pub fn to_address(&self) -> Option<Address> {
```

Comment:
nit: `to_evm_address` is more future-proof as we may rename `AccountOwner` in the future.

Reply:
Yes, that is a better name.